src/i18n/locale-fr.json

```diff
@@ -119,6 +119,31 @@
             "content": "Sélectionner les nœuds à retirer du cluster existant"
         }
     },
+    "acl_management": {
+        "rulestable": {
+            "column": {
+                "index": "#",
+                "reorder": "",
```

Comment:
What is this for?

Reply:
The label for the reorder actions column :)
Maybe I should remove it